src/main/webapp/jokes.jsp

```diff
@@ -0,0 +1,35 @@
+<%-- 
```

Comment:
Are you sure we should be using jsp? I think we're supposed to run with plain html-files and javascript for everything in the frontend?

Reply:
i'll swap it over :)